src/sinks/datadog/logs/sink.rs

```diff
@@ -91,6 +100,11 @@ pub struct LogSink<S> {
     batch_settings: BatcherSettings,
     /// The protocol name
     protocol: String,
+    /// When enabled this sink will normalize events to conform to the datadog agent standard. This
```

Comment:
```suggestion
    /// When enabled this sink will normalize events to conform to the Datadog Agent standard. This
```

Can we link to the docs here?